libs/webb-ui-components/src/components/CheckBox/Checkbox.tsx

```diff
@@ -45,15 +45,16 @@ export const CheckBox: React.FC<CheckBoxProps> = (props) => {
   } = props;
 
   const inputClsx =
-    'form-checkbox peer transition-none bg-mono-0 w-[18px] h-[18px] rounded border border-mono-100 outline-none dark:bg-mono-180';
+    'form-checkbox peer transition-none bg-mono-0 w-[18px] h-[18px] rounded border-2 border-mono-100 outline-none dark:bg-mono-180 focus:ring-offset-0 focus:ring-0';
 
   const inputHoverClsx =
     'enabled:hover:shadow-sm enabled:hover:shadow-blue-10 dark:hover:shadow-none';
 
-  const inputCheckedClsx = 'checked:bg-blue-70 dark:checked:bg-blue-50';
+  const inputCheckedClsx =
+    'checked:!bg-blue-50 checked:hover:bg-blue-50 checked:active:!bg-blue-60 dark:checked:active:!bg-blue-40';
```

Comment:
I'm curious why you added `!important`, as it complicates component customization.